corehq/apps/locations/fixtures.py

```diff
@@ -214,6 +214,26 @@ def should_sync_flat_fixture(project):
 )
 
 
+class LocationGroupFixtureProvider(FixtureProvider):
```

Comment:
This has a few assumptions baked in that I'd like to check:

Only location groups associated with users' primary locations matter.  If a user is assigned to multiple locations, only one will contribute to this fixture.

The "nested groups" nature of locations doesn't apply here.  That is, if a descendant of your location is in a group, you won't (necessarily) see the related locations.

Similarly, if your location has related locations, you'll see those, but you won't see their descendants or ancestors.

Reply:
https://github.com/dimagi/commcare-hq/pull/21349/commits/bb23878d009fb6cbacf7975bcea3f419af866afc